flink-core/src/main/java/org/apache/flink/configuration/HighAvailabilityOptions.java

```diff
@@ -100,6 +100,45 @@
 			.defaultValue("/flink")
 			.withDeprecatedKeys("recovery.zookeeper.path.root");
 
+	public static final ConfigOption<String> HA_ZOOKEEPER_NAMESPACE =
+			key("high-availability.zookeeper.path.namespace")
+			.noDefaultValue();
+
+	public static final ConfigOption<String> HA_ZOOKEEPER_LATCH_PATH =
+			key("high-availability.zookeeper.path.latch")
+			.defaultValue("/leaderlatch")
+			.withDeprecatedKeys("recovery.zookeeper.path.latch");
+
+	/** ZooKeeper root path (ZNode) for job graphs. */
+	public static final ConfigOption<String> HA_ZOOKEEPER_JOBGRAPHS_PATH =
+			key("high-availability.zookeeper.path.jobgraphs")
+			.defaultValue("/jobgraphs")
+			.withDeprecatedKeys("recovery.zookeeper.path.jobgraphs");
+
+	public static final ConfigOption<String> HA_ZOOKEEPER_LEADER_PATH =
+			key("high-availability.zookeeper.path.leader")
+			.defaultValue("/leader")
+			.withDeprecatedKeys("recovery.zookeeper.path.leader");
+
+	/** ZooKeeper root path (ZNode) for completed checkpoints. */
+	public static final ConfigOption<String> HA_ZOOKEEPER_CHECKPOINTS_PATH =
+			key("high-availability.zookeeper.path.checkpoints")
+			.defaultValue("/checkpoints")
+			.withDeprecatedKeys("recovery.zookeeper.path.checkpoints");
+
+	/** ZooKeeper root path (ZNode) for checkpoint counters. */
+	public static final ConfigOption<String> HA_ZOOKEEPER_CHECKPOINT_COUNTER_PATH =
```

Comment:
Thank you for your reply. In addition to the zookeeper options, there're some other options such as `HA_STORAGE_PATH`, `HA_JOB_MANAGER_PORT_RANGE`. I think the `HA_` prefix of them should be removed, otherwise the style may not be consistent. What do you think?